calm/control-example/pre-prod-review-configuration.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "control-specification.json",
+  "$schema": "pre-prod-review-specification.json",
   "$id": "control-evidence.json",
```

Comment:
Given this is an JSON instance document, not a schema, I don't think we need an id?